evm/src/main/java/org/hyperledger/besu/evm/code/OpcodesV1.java

```diff
@@ -294,34 +299,77 @@ private OpcodesV1() {
 
   static long[] validateAndCalculateJumpDests(final Bytes code) {
```

Comment:
Yes, this can probably be refactored somewhere else, but I didn't want to go too far with the changes for this PR. I would say that we can let EOF to settle down a bit, keep an eye for its final form and then propose some movements around it